### DISCUSSION TITLE
Why chakra 3 is not following the same naming convention as ArkUI?

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/chakra-ui#9002] [Q&A] [answered]
As far as I have seen on ArkUI and P-ARK UI, they follow another naming while Chakra is providing another, at least based on what I am seeing on the Docs.

> Ark UI
````tsx
import { Menu } from '@ark-ui/react/menu'

export const Basic = () => (
  <Menu.Root>
    <Menu.Trigger>
      Open menu <Menu.Indicator>➡️</Menu.Indicator>
    </Menu.Trigger>
    <Menu.Positioner>
      <Menu.Content>
        <Menu.Item value="react">React</Menu.Item>
        <Menu.Item value="solid">Solid</Menu.Item>
        <Menu.Item value="vue">Vue</Menu.Item>
      </Menu.Content>
    </Menu.Positioner>
  </Menu.Root>
)

````

> Chakra
````tsx
<MenuRoot>
  <MenuTrigger />
  <MenuContent>
    <MenuItem value="..." />
  </MenuContent>
</MenuRoot>
````

@segunadebayo

**Comment by segunadebayo** [comment, marked as answer]
Park UI will be updated pretty soon to match Chakra. The goal is to reduce the lines of code needed to get a component working and looking good.

The MenuContent in Chakra encapsulates the MenuContent+Position in Ark. It is just a quick win for most Chakra folks. Remember, we have to make the migration from v2 somewhat intuitive.
